community-meetings/2021-05-11.md

```diff
@@ -22,10 +23,60 @@ Details on our ClusterAPI work so far and future plans
 ## Community Q&A
 Open Q&A / discussion
 
----
 
-# Minutes
-The minutes will be published here after the call concluded.
+# Call Minutes
+
+The meeting largely followed the [slide deck](2021-05-11-slides.pdf). After the presentation, community participants raised a total of 3 questions.
+
+1. The Flatcar team, Thilo, Sayan, Marga, Dongsu, Mathieu, Andy, Kai, and Iago introduce themselves.
+2. Andy briefly addresses the future of Flatcar following Kinvolk's acquisition by Microsoft.
+   - " [...] we want to assure the Flatcar community that Microsoft and the Kinvolk team will continue to collaborate with the larger Flatcar community on the evolution of Flatcar Container Linux.” - Brendan Burns, Microsoft
+   - " “This will not be a replay of the movie you’ve seen before. In fact, we and Microsoft are committed to doubling down on the Flatcar community: we want to expand the universe of partners, contributors, and users, to ensure a vibrant, successful and sustainable long-term future for Flatcar as a truly open, community-driven project.” - Chris Kühl, Kinvolk
```

Comment:
```suggestion
   - “This will not be a replay of the movie you’ve seen before. In fact, we and Microsoft are committed to doubling down on the Flatcar community: we want to expand the universe of partners, contributors, and users, to ensure a vibrant, successful and sustainable long-term future for Flatcar as a truly open, community-driven project.” - Chris Kühl, Kinvolk
```